projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/sample/Sample.java

```diff
@@ -0,0 +1,88 @@
+package life.qbic.projectmanagement.domain.project.sample;
+
+import jakarta.persistence.Embedded;
+import jakarta.persistence.EmbeddedId;
+import jakarta.persistence.Entity;
+import life.qbic.projectmanagement.domain.project.experiment.BiologicalReplicateId;
+import life.qbic.projectmanagement.domain.project.experiment.ExperimentId;
+
+/**
+ * <b>Sample Entity</b>
+ * <p>
+ * A sample represents the physical sample from an experiment that has been collected and needs to
+ * be prepared for measurement in one of QBiC's partner facilities.
+ * <p>
+ * A sample needs to be registered and assigned to one existing sample {@link Batch}, before it can
+ * be prepared for shipment to the measurement facility.
+ *
+ * @since 1.0.0
+ */
+@Entity(name = "sample")
+public class Sample {
+
+  @EmbeddedId
+  private SampleId id;
+
+  private String label;
+
+  @Embedded
+  private BatchId assignedBatch;
+
+  private Long experimentalGroupId;
+
+  @Embedded
+  private ExperimentId experimentId;
+
+  @Embedded
+  private SampleOrigin sampleOrigin;
+
+  @Embedded
```

Comment:
Is there a logical ordering of the fields? If not I would welcome a logical ordering or alphabetic.